config/generic-mellow-fly-d5.cfg

```diff
@@ -0,0 +1,143 @@
+# "USB to CAN bus bridge (USB on PA11/PA12)"
+
+# See docs/Config_Reference.md for a description of parameters.
+
+[mcu]
+serial: /dev/serial/by-id/usb-klipper-stm32f072xx-1234567890987654321 # Replace with your serial address here, see docs/FAQ.md#wheres-my-serial-port for more
+#canbus_uuid: 11aa22bb33cc # Replace with your canbus uuid here, see docs/CANBUS.md for more
+
+[temperature_sensor Fly-D5] # Mainboard Temperature
+sensor_type: temperature_mcu
+
+[printer]
+kinematics: corexy # Replace with your machine's kinematic!(e.g. cartesian/delta)
+max_velocity: 350
+max_accel: 10000
+max_z_velocity: 10
+max_z_accel: 200
+
+[extruder]
+step_pin: PC5
+dir_pin: !PC4
+enable_pin: !PB0
+microsteps: 64
+nozzle_diameter: 0.4
+filament_diameter: 1.75
+rotation_distance: 23.310912
+gear_ratio: 50:10
+heater_pin: PC6
+sensor_pin: PC1
+sensor_type: Generic 3950
+control: pid
+pid_Kp: 10
+pid_Ki: 10
+pid_Kd: 10
+min_temp: 0
+max_temp: 300
+
+#[tmc2209 extruder] # Uncomment this section if using a TMC2209 stepper driver, replace it with your TMC model. Applies to the other [tmc2209] section.
+#uart_pin: PA7
+#interpolate: True
+#run_current: 0.8
+#stealthchop_threshold: 0
+
+[stepper_x]
+step_pin: PC15
+dir_pin: PC14
+enable_pin: !PC2
+microsteps: 64
+rotation_distance: 40   # See docs/Rotation_Distance.md
+endstop_pin:^PB4
+position_endstop: 0
+position_max: 200
+
+#[tmc2209 stepper_x]
+#uart_pin: PC13
+#interpolate: True
+#run_current: 1.2
+#stealthchop_threshold: 0
+
+[stepper_y]
+step_pin: PA1
+dir_pin: PA0
+enable_pin: !PA2
+microsteps: 64
+rotation_distance: 40 # Same as above
+endstop_pin: ^PB3
+position_endstop: 0
+position_max: 200
+
+#[tmc2209 stepper_y]
+#uart_pin: PC3
+#interpolate: True
+#run_current: 1.2
+#stealthchop_threshold: 0
+
+[stepper_z]
+step_pin: PA5
+dir_pin: PA4
+enable_pin: !PA6
+microsteps: 64
+rotation_distance: 8
+endstop_pin: ^PD2
+position_endstop: 0
+position_max: 200
+
+#[tmc2209 stepper_z]
+#uart_pin: PA3
+#interpolate: True
+#run_current: 0.8
+#stealthchop_threshold: 0
+
+#[stepper_z1] # Uncomment this when using dual Z motors. Pins are the "E1" motor pins on the board.
+#step_pin: PB10
+#dir_pin:  PB2
+#enable_pin: !PB11
+#microsteps: 64
+#rotation_distance: 8
+#position_endstop: 0
+#position_max: 200
+
+#[tmc2209 stepper_z1]
+#uart_pin: PB1
+#interpolate: True
+#run_current: 0.8
+#stealthchop_threshold: 0
+
+#[bltouch]
+#sensor_pin: ^PB5
+#control_pin:PA8
+
+#[safe_z_home] #When using a probe for Z-endstop, it is recommended to use [safe_z_home] section to home the Z axis at the centre of your print bed.
+
+[fan]
+pin: PC8
+
+[heater_fan Heatbreak_Fan]
+pin:PC9
+max_power:1
+shutdown_speed:0
+
+[heater_bed]
+heater_pin: PC7
+sensor_pin: PC0
+sensor_type: Generic 3950
+control: pid #Run "PID_CALIBRATE HEATER=heater_bed TARGET=270" to calibrate flowing pid control values before print! They are just placeholders!
+pid_Kp: 10
+pid_Ki: 10
+pid_Kd: 10
+min_temp: 0
+max_temp: 100
```

Comment:
Also, for sake of completeness: The `[board_pins]` for the EXP ports are missing. See https://github.com/Klipper3d/klipper/blob/master/config/generic-bigtreetech-manta-m5p.cfg